authentik/policies/geoip/views.py

```diff
@@ -0,0 +1,20 @@
+from django_countries import countries
+from django_countries.serializer_fields import CountryField
+from rest_framework import serializers
+from rest_framework.generics import ListAPIView
+from rest_framework.permissions import AllowAny
+
+
+class DetailedCountrySerializer(serializers.Serializer):
+    code = CountryField()
+    name = serializers.CharField()
+
+
+class ISO3166View(ListAPIView):
```

Comment:
I think there is an argument for putting this in `api/` (or api.py) as well, for consistency with how we do things elsewhere.

Reply:
That makes sense. This used to be much more convoluted, but I should have moved it when I switched it to declarative style. Done!